src/pages/ClusterPage/explore.js

```diff
@@ -17,13 +17,16 @@ export default class ExploreCluster extends Component {
 		const cluster = props.location.state
 			? props.location.state.cluster
 			: null;
-
+		const redirectToBilling = props.location.state
```

Comment:
this is just a copy of the above statement, what is the requirement of this variable?

Reply:
Actually this is for another use case. It's like in dashboard, when you click the red "upgrade now" button in header, you need to redirected into arc's cluster billing page.

Reply:
got it 👍